### PR TITLE
TASK1: rock-paper-scissors game

### PR DESCRIPTION
**Pull request by d-serj**
Rock-paper-scissors game.

To play the game enter your sign: r - for "Rock", p - for "Paper", s - for "Scissors".
Press Ctrl+D or Ctrl+Z to exit the game

### CODE REVIEW

File: 01_git/Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+CC_SOURCES=main.c
+
+rock_paper_scissors:
+	$(CC) $(CC_FLAGS) $(CC_SOURCES) -o$@
+
+all: rock_paper_scissors
+
+clean:
+	rm rock_paper_scissors
```
Comment (serhii-perederii) [inline 01_git/Makefile:12]: Add newline at the end of the file

Comment (serhii-perederii) [reply 01_git/Makefile:12]: Think about automatic cleaning the whole directory from build artifacts

**Review by serhii-perederii — CHANGES_REQUESTED** [review]

File: 01_git/Makefile (hunk 1 above)
Comment (d-serj) [reply 01_git/Makefile]: Automatic cleaning the whole directory from build artifacts was added.

Comment (d-serj) [reply 01_git/Makefile]: Newline at the end of the file was added.

**Comment by d-serj** [comment]
@serhii-perederii I have fixed your remarks. Could you please review it once again?

**Review by serhii-perederii — APPROVED** [review]

**Pull request merged**